### ISSUE TITLE
Import tasks from github issues

### ISSUE THREAD

**Issue by tj** [tj/pomo#11]

**Comment by cj** [comment]
I just tried this feature, the errors I'm getting:

```
09:19 ~/dropbox/sd/acd/auto (live/master *1)$ pomo import softwaredevs ACD-Auto
/Users/cj/.rvm/rubies/ruby-1.9.3-p0/lib/ruby/site_ruby/1.9.1/rubygems/dependency.rb:247:in `to_specs': Could not find octopi (>= 0.2.1) amongst [RedCloth-4.2.9, abstract-1.0.0, actionmailer-3.2.0, actionmailer-3.1.3, actionmailer-3.1.2, actionmailer-3.1.1, actionmailer-3.1.0, actionmailer-3.0.11, actionmailer-3.0.0, actionpack-3.2.0, actionpack-3.1.3, actionpack-3.1.2, actionpack-3.1.1, actionpack-3.1.0, actionpack-3.0.11, actionpack-3.0.0, activemodel-3.2.0, activemodel-3.1.3, activemodel-3.1.2, activemodel-3.1.1, activemodel-3.1.0, activemodel-3.0.11, activemodel-3.0.0, activerecord-3.2.0, activerecord-3.1.3, activerecord-3.1.2, activerecord-3.1.1, activerecord-3.1.0, activerecord-3.0.11, activerecord-3.0.0, activeresource-3.2.0, activeresource-3.1.3, activeresource-3.1.2, activeresource-3.1.1, activeresource-3.1.0, activeresource-3.0.11, activeresource-3.0.0, activesupport-3.2.0, activesupport-3.1.3, activesupport-3.1.2, activesupport-3.1.1, activesupport-3.1.0, activesupport-3.0.11, activesupport-3.0.0, addressable-2.2.6, airbrake-3.0.9, akami-1.0.0, albino-1.3.3, ansi-1.4.1, ansi-1.3.0, archive-tar-minitar-0.5.2, arel-3.0.0, arel-2.2.1, arel-2.0.10, arel-1.0.1, bcrypt-ruby-3.0.1, bourbon-1.2.0, bson-1.5.2, bson_ext-1.5.2, builder-3.0.0, builder-2.1.2, bundler-1.0.21, cancan-1.6.7, capistrano-2.9.0, capistrano-ext-1.2.1, capybara-1.1.2, capybara-webkit-0.8.0, capybara-webkit-0.7.2, childprocess-0.3.0, childprocess-0.2.5, childprocess-0.2.4, childprocess-0.2.3, chronic-0.6.6, chunky_png-1.2.5, classifier-1.3.3, client_side_validations-3.1.3, cocaine-0.2.1, coffee-rails-3.2.1, coffee-rails-3.1.1, coffee-script-2.2.0, coffee-script-source-1.2.0, coffee-script-source-1.1.3, coffee-script-source-1.1.2, columnize-0.3.6, commander-4.0.7, commonjs-0.2.0, compass-0.12.alpha.4, compass-0.11.7, crack-0.3.1, daemon_controller-0.2.6, daemons-1.1.4, database_cleaner-0.7.1, database_cleaner-0.7.0, devise-1.5.3, devise-1.5.2, diff-lcs-1.1.3, directory_watcher-1.4.1, enumerated_attribute-0.2.16, erubis-2.7.0, erubis-2.6.6, escape-0.0.4, eventmachine-0.12.10, exception_notification-2.5.2, exceptional-2.0.32, execjs-1.3.0, execjs-1.2.13, execjs-1.2.12, execjs-1.2.9, factory_girl-2.5.0, factory_girl-2.3.2, factory_girl_rails-1.6.0, factory_girl_rails-1.4.0, fakeweb-1.3.0, fast-stemmer-1.0.0, fastercsv-1.5.4, fastthread-1.0.7, ffi-1.0.11, foreman-0.31.0, formtastic-2.0.2, fssm-0.2.8.1, gravtastic-3.2.6, growl-1.0.3, growl_notify-0.0.3, guard-1.0.0, guard-0.10.0, guard-0.9.3, guard-0.8.8, guard-resque-0.0.3, guard-rspec-0.6.0, guard-rspec-0.5.10, guard-rspec-0.5.9, guard-spork-0.5.1, guard-spork-0.5.0, guard-spork-0.4.1, gyoku-0.4.4, haml-3.1.4, haml-3.1.3, has_scope-0.5.1, heroku-2.16.1, highline-1.6.8, highline-1.5.2, hike-1.2.1, hpricot-0.8.6, httparty-0.8.1, httpi-0.9.5, i18n-0.6.0, i18n-0.5.0, i18n-0.4.2, inherited_resources-1.3.0, inherited_resources-1.2.2, jekyll-0.11.2, journey-1.0.0, jquery-rails-2.0.0, jquery-rails-1.0.19, jquery-rails-1.0.16, json-1.6.5, json-1.6.4, json-1.6.3, json-1.6.1, json-1.5.4, json_pure-1.6.3, jwt-0.1.4, kaminari-0.13.0, kaminari-0.12.4, kramdown-0.13.4, launchy-2.0.5, less-2.0.9, less-2.0.8, less-rails-2.1.3, less-rails-2.1.2, less-rails-2.1.1, letter_opener-0.0.2, libv8-3.3.10.4-x86_64-darwin-11, libv8-3.3.10.2, linecache19-0.5.12, liquid-2.3.0, macaddr-1.5.0, mail-2.4.1, mail-2.3.0, mail-2.2.19, maruku-0.6.0, meta_programming-0.2.2, meta_search-1.1.1, metaclass-0.0.1, mime-types-1.17.2, mocha-0.10.3, mocha-0.10.0, money-4.0.1, mongo-1.5.2, mongo_mapper-0.9.2, mongoid-2.0.2, multi_json-1.0.4, multi_json-1.0.3, multi_xml-0.4.1, mysql2-0.3.11, mysql2-0.2.18, net-scp-1.0.4, net-sftp-2.0.5, net-ssh-2.2.1, net-ssh-2.1.4, net-ssh-gateway-1.1.0, nokogiri-1.5.0, nori-1.0.2, orm_adapter-0.0.5, paperclip-2.4.5, passenger-3.0.11, pg-0.12.0, pg-0.11.0, plucky-0.3.8, polyamorous-0.5.0, polyglot-0.3.3, pomo-1.0.1, posix-spawn-0.3.6, powder-0.1.7, pr_geohash-1.0.0, rack-1.4.1, rack-1.4.0, rack-1.3.6, rack-1.3.5, rack-1.2.5, rack-cache-1.1, rack-cache-1.0.3, rack-mount-0.8.3, rack-mount-0.6.14, rack-protection-1.2.0, rack-protection-1.1.4, rack-ssl-1.3.2, rack-test-0.6.1, rack-test-0.5.7, rails-3.2.0, rails-3.1.3, rails-3.1.2, rails-3.1.1, rails-3.1.0, rails-3.0.11, rails-3.0.0, railties-3.2.0, railties-3.1.3, railties-3.1.2, railties-3.1.1, railties-3.1.0, railties-3.0.11, railties-3.0.0, rake-0.9.2.2, rake-0.8.7, rake-compiler-0.7.9, rb-appscript-0.6.1, rb-fchange-0.0.5, rb-fsevent-0.4.3.1, rb-inotify-0.8.8, rdiscount-1.6.8, rdoc-3.12, rdoc-3.11, redis-2.2.2, redis-namespace-1.0.3, redisk-0.2.2, responders-0.6.4, resque-1.19.0, resque-exceptional-0.1.0, resque-retry-0.2.2, resque-scheduler-1.9.9, resque-status-0.2.4, resque_mailer-2.0.2, rest-client-1.6.7, rest-client-1.6.1, role_model-0.7.0, rsolr-1.0.6, rspec-2.8.0, rspec-2.8.0.rc2, rspec-2.8.0.rc1, rspec-2.7.0, rspec-core-2.8.0, rspec-core-2.8.0.rc2, rspec-core-2.8.0.rc1, rspec-core-2.7.1, rspec-expectations-2.8.0, rspec-expectations-2.8.0.rc2, rspec-expectations-2.8.0.rc1, rspec-expectations-2.7.0, rspec-mocks-2.8.0, rspec-mocks-2.8.0.rc2, rspec-mocks-2.8.0.rc1, rspec-mocks-2.7.0, rspec-rails-2.8.1, rspec-rails-2.8.0.rc2, rspec-rails-2.8.0.rc1, ruby-debug-base19-0.11.25, ruby-debug19-0.11.6, ruby_core_source-0.1.5, ruby_gntp-0.3.4, rubyzip-0.9.5, rufus-scheduler-2.0.16, rufus-scheduler-2.0.14, sass-3.1.12, sass-3.1.11, sass-3.1.10, sass-rails-3.2.3, sass-rails-3.1.5, sass-rails-3.1.4, savon-0.9.7, selenium-webdriver-2.17.0, selenium-webdriver-2.16.0, selenium-webdriver-2.15.0, sendgrid-1.0.1, shotgun-0.9, showoff-io-0.3.1, simple_form-1.5.2, sinatra-1.3.2, sinatra-1.3.1, spork-0.9.0, spork-0.9.0.rc9, sprockets-2.1.2, sprockets-2.0.3, sqlite3-1.3.5, sqlite3-ruby-1.3.3, stripe-1.5.25, sunspot-1.3.0, sunspot_rails-1.3.0, syntax-1.0.0, systemu-2.4.2, term-ansicolor-1.0.7, therubyracer-0.9.9, therubyracer-0.9.8, thin-1.3.1, thor-0.14.6, tilt-1.3.3, timecop-0.3.5, treetop-1.4.10, turn-0.8.3, turn-0.8.2, twilio-rb-2.1.3, tzinfo-0.3.31, tzinfo-0.3.30, uglifier-1.2.2, uglifier-1.2.1, uglifier-1.2.0, uglifier-1.1.0, uglifier-1.0.4, uuid-2.3.4, vagrant-0.8.10, vcr-1.11.3, vegas-0.1.8, virtualbox-0.9.2, warden-1.1.0, wasabi-2.0.0, webmock-1.7.10, webmock-1.7.8, xpath-0.1.4, yajl-ruby-1.1.0, yajl-ruby-1.0.0] (Gem::LoadError)
    from /Users/cj/.rvm/rubies/ruby-1.9.3-p0/lib/ruby/site_ruby/1.9.1/rubygems/dependency.rb:256:in `to_spec'
    from /Users/cj/.rvm/rubies/ruby-1.9.3-p0/lib/ruby/site_ruby/1.9.1/rubygems.rb:1210:in `gem'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/pomo-1.0.1/bin/pomo:71:in `block (2 levels) in <top (required)>'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/command.rb:180:in `call'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/command.rb:180:in `call'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/command.rb:155:in `run'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/runner.rb:402:in `run_active_command'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/runner.rb:66:in `run!'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/delegates.rb:11:in `run!'
    from /Users/cj/.rvm/gems/ruby-1.9.3-p0@global/gems/commander-4.0.7/lib/commander/import.rb:10:in `block in <top (required)>'
```

**Comment by stephenmckinney** [comment]
Fixed in 2.0 release.
